sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -1455,41 +1427,4 @@
       </build>
     </profile>
   </profiles>
-
-  <modules>
```

Comment:
Have the aggregate-reports been fixed where this is no longer needed?

Reply:
Aggregate reports runs against the root ```./pom.xml``` file and uses that module list. The parent POM isn't even in the graph directly except by parent references (which don't cause module evaluation).

Reply:
... unless I misunderstand the question/concern :)

Reply:
Couldn't we just use this goal?

http://maven.apache.org/plugins/maven-checkstyle-plugin/checkstyle-aggregate-mojo.html

Then do something similar to what we do with the code coverage reports and have a module which runs this goal and does the aggregation?

Reply:
So we do have that goal configured in the client parent POM, but the configuration isn't allowed to be inherited by children POMs. I think we could allow it to be inherited to resolve the Checkstyle aggregate generation issue.